lib/config.js

```diff
@@ -37,6 +37,11 @@ var envFromShell = process.env.NODE_ENV;
 
 var env = envFromBrowser || envFromShell || DEFAULT_ENV;
 
+if (!env.match(/^(production|staging|cam)$/)) { 
+  throw 'Panoptes Javascript Client Error: Invalid Environment; ' +
```

Comment:
This throws a string, which is generally unexpected. Try `throw new Error('...')`.